.env.template

```
@@ -68,32 +68,36 @@ AUTH_USE_AUTH=false
-# LLM_DEFAULT_MAX_TOKENS=2500
+# LLM_EMBEDDING_BASE_URL=http://127.0.0.1:8100  # For local provider or provider-specific override
+# LLM_EMBEDDING_MODEL=mlx-community/Qwen3-Embedding-0.6B-4bit-DWQ
+# LLM_EMBEDDING_API_KEY=***
+# LLM_DEFAULT_MAX_TOKENS=***
 # LLM_MAX_TOOL_OUTPUT_CHARS=10000
 # LLM_MAX_MESSAGE_CONTENT_CHARS=2000
 
 # =============================================================================
 # Deriver (Background Worker)
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)